macros/ui/niceTables.pl

```diff
@@ -369,6 +369,23 @@ =head2 Options for ROWS
 Override table's overall vertical alignment for this row.  Can be C<'top'>, C<'middle'>,
 or C<'bottom'>.
 
+=item C<rows =E<gt> 2D array reference>
```

Comment:
FYI: If you don't want to use the E<gt> format, this can be done with `C<< rows => 2D array reference >>`.  Note there needs to be a space after the opening `<<` and before the closing `>>`.

Reply:
I agree with @pstaabp.  I have preferred that way of doing this since it is more readable when you are looking at the code.  Of course in the generated POD it is equivalent.